packages/manager/modules/pci/src/projects/project/instances/add/add.controller.js

```diff
@@ -151,10 +151,17 @@ export default class PciInstancesAddController {
     this.floatingIps = null;
     this.addons = [];
     this.getSmallestGatewayInfo();
-    this.defaultFloatingIp = this.getProductCatalog;
+    this.defaultFloatingIp = this.getProductCatalog.defaultFloatingIp;
+    this.publicIpProducts = this.getProductCatalog.publicIpProducts;
     this.isIpLoading = false;
   }
 
+  getPublicIpPriceInformation() {
+    return this.publicIpProducts
+      .find((product) => product.planCode.endsWith(this.model.datacenter.name))
+      .pricings.find(({ type }) => type === 'consumption').price;
```

Comment:
no possibility to not find any price ? If yes, you should add check nullity condition

Reply:
Done